lib/shim.c

```diff
@@ -216,6 +216,11 @@ coding_game_service_app_integration_controller_init(CodingGameServiceAppIntegrat
   }
 }
 
+CodingGameServiceAppIntegrationController *
+coding_game_service_app_integration_controller_new () {
+  return g_object_new(CODING_GAME_SERVICE_TYPE_APP_INTEGRATION_CONTROLLER);
```

Comment:
- you need to `NULL`-terminate the parameters to `g_object_new()`
- you should put `void` as argument, both here and in the header file declaration